pages/star-search/index.tsx

```diff
@@ -634,7 +633,11 @@ function Chatbox({ message, userId }: { message: StarSearchChat; userId?: number
   if (typeof message.content == "string") {
     // Breaking all words so that the rendered markdown doesn't overflow the container
     // in certain cases where the content is a long string.
-    content = <Markdown className="break-all">{message.content}</Markdown>;
+    content = (
+      <Markdown remarkPlugins={[remarkGfm]} className="break-words">
```

Comment:
It renders the list to the correct HTML, just gotta style it with Tailwind. Just add `prose` to the className for `Markdown`

![image](https://github.com/open-sauced/app/assets/20603494/5c63fc2f-a437-4fe9-b4aa-4a81381ed09f)

Reply:
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExeGhqdWZjMHcxbjd0dGNhb2YyYzBlMzUyMnFzcjVjMmRhZDMzOW9rYSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3WCNY2RhcmnwGbKbCi/giphy.gif)

Oh yeah, this looks so much better 👏🏼 👏🏼 good find @zeucapua 🏅🏅🏅🏅🏅🏅🏅🏅🏅🏅🏅🏅🏅🏅🏅🏅